modules/sql-engine/src/integrationTest/java/org/apache/ignite/internal/sql/engine/ItAlterTableDdlTest.java

```diff
@@ -152,11 +150,6 @@ public void testNullableColumn() {
     public void testDropAndAddColumnsAllTypes() {
         List<NativeType> allTypes = SchemaTestUtils.ALL_TYPES;
 
-        Set<NativeTypeSpec> unsupportedTypes = Set.of(
-                // TODO https://issues.apache.org/jira/browse/IGNITE-18431
-                NativeTypeSpec.BITMASK
```

Comment:
what about other places where [IGNITE-18431](https://issues.apache.org/jira/browse/IGNITE-18431) is mentioned?
Should we mention that all such places should be cleared in the `follower` ticket?